imports/plugins/included/product-admin/client/blocks/VariantTaxForm.js

```diff
@@ -0,0 +1,103 @@
+import React from "react";
```

Comment:
Technically this block should be added by the core `taxes` plugin because that's what extends the product schema to add these fields. But if it isn't easy to move it right now, it's fine to punt that.

Reply:
I'd rather leave it for now so this change can be isolated in its own PR.

Reply:
Added a ticket here: https://github.com/reactioncommerce/reaction/issues/5103